### PR TITLE
Feature/5409-favorites-list-material-component

### PR DESCRIPTION
**Pull request by jeppekroghitk**
#### Link to issue

https://platform.dandigbib.org/issues/5409

#### Description

Created new styling class component for favorites list, displaying 4 favorites in a row, adjusting to 2 per row on smaller screens. Included preview component for storybook.

#### Screenshot of the result

Screenshot 2023-03-13 at 09 31 33

#### Checklist

- [ ] My complies with [our coding guidelines](../docs/code_guidelines.md).
- [ ] My code passes our static analysis suite. If not then I have added a comment explaining why this change should be exempt from the code standards and process.
- [ ] My code passes our continuous integration process. If not then I have added a comment explaining why this change should be exempt from the code standards and process.

#### Additional comments or questions

If you have any further comments or questions for the reviewer them please add them here.

### CODE REVIEW

File: src/stories/Library/favorites-list-material-component/favorites-list-material-component.scss [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    text-align: center;
+    padding: $s-lg;
+
+    @include breakpoint-m {
+      padding-bottom: $s-6xl;
+    }
+  }
+  &__title--left {
+    text-align: left;
+  }
```
Comment (spaceo) [inline src/stories/Library/favorites-list-material-component/favorites-list-material-component.scss:30]:
```suggestion
  &__title {
    text-align: center;
    padding: $s-lg;

     &--left {
        text-align: left;
      }
  
    @include breakpoint-m {
      padding-bottom: $s-6xl;
    }
  }
 
```

File: src/stories/Library/favorites-list-material-component/favorites-list-material-component.scss [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+  height: 350px;
+
+  &--bright {
+    border: 1px solid $c-global-tertiary-1;
+  }
+
+  &__favourite {
+    position: absolute;
+    top: 10px;
+    right: 10px;
```
Comment (spaceo) [inline src/stories/Library/favorites-list-material-component/favorites-list-material-component.scss:64]: Is it possible to use spacing variables here?

File: src/stories/Library/favorites-list-material-component/favorites-list-material-component.scss [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+  }
+
+  &__cover-container {
+    display: flex;
+    justify-content: center;
+    width: 100%;
+    margin-top: 10px;
```
Comment (spaceo) [inline src/stories/Library/favorites-list-material-component/favorites-list-material-component.scss:71]: Is it possible to use a spacing variable here?

File: src/stories/Library/favorites-list-material-component/favorites-list-material-component.scss [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+  }
+
+  &__meta {
+    position: absolute;
+    left: 10px;
+    bottom: 10px;
```
Comment (spaceo) [inline src/stories/Library/favorites-list-material-component/favorites-list-material-component.scss:77]: Is it possible to use spacing variables here?

File: src/stories/Library/favorites-list-material-component/favorites-list-material-component.tsx [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+                  className={`icon-favourite icon-favourite${
+                    !bright ? "--bright" : ""
+                  } ${
+                    filled
+                      ? `icon-favourite--${!bright ? "bright-" : ""}filled`
+                      : ""
+                  }`}
+                />
+              </div>
+            </div>
```
[the remark this answers is not in the document]
Comment (jeppekroghitk) [reply src/stories/Library/favorites-list-material-component/favorites-list-material-component.tsx:54]: As there is a mix between whole classes, im not sure a function would ease the overview much.

The already merged recommender uses the same approach, see https://github.com/danskernesdigitalebibliotek/dpl-design-system/pull/190/files#diff-ec62aa3816e4093dec9c5c3c0663531350b6deafff378e048c9118eb8664264f
so i do not see this being a change request as much as a comment, but i added a TODO for it.

**Review by spaceo — CHANGES_REQUESTED** [review]
@jeppekroghitk Sorry if I was not precise enough in my review but when I talked about spacing variables I actually meant these:
https://github.com/danskernesdigitalebibliotek/dpl-design-system/blob/873c3a33a08bc91202e72a04ab7f1d958b20c770/src/stories/Library/spacing/spacing.scss#L17-L27
The idea is that by using them it will be possible to adjust the whole design if we find eg. the medium spacing should be smaller or greater than the present value. Then the whole design will change accordingly.

**Pull request merged**